lib/honeybadger.ex

```diff
@@ -160,26 +158,25 @@ defmodule Honeybadger do
     Supervisor.start_link(children, strategy: :one_for_one)
   end
 
-  @doc false
-  def stop(_state) do
-    :error_logger.delete_report_handler(Honeybadger.Logger)
-
-    :ok
-  end
-
   @doc """
   Send an exception notification, if reporting is enabled.
 
   This is the primary way to do manual error reporting and it is also used
   internally to deliver logged errors.
 
+  ## Stacktrace
+
+  Accessing the stacktrace outside of a rescue/catch is deprecated. Notifiations should happen
```

Comment:
Do people currently use `Honeybadger.notify` outside of rescue/catch blocks? If so, do we need to let them know that they should stop?

Reply:
People may. Using `Honeybadger.notify/3` will still work, there just won't be a stacktrace included. For what it's worth, Sentry and Rollbax work the same way.